src/lib/translators/mongo.ts

```diff
@@ -105,23 +105,42 @@ Object.assign(Mongo36Translator.prototype, mapper);
  *
  * @returns the list of simplified mongo steps
  */
-function simplifyMongoPipeline(mongoSteps: Array<MongoStep>): Array<MongoStep> {
+export function _simplifyMongoPipeline(mongoSteps: Array<MongoStep>): Array<MongoStep> {
+  let merge: boolean = true;
```

Comment:
In such cases, you should let typescript infer type automatically, you don't have to specify `boolean` explicitly.

```suggestion
  let merge = true;
```